src/components/translations/en.js

```diff
@@ -1,8 +1,40 @@
-export let messages_en= {
```

Comment:
Please, move _translations_ folder one level up (replace from '_components_' to '_app_' folder).